.github/workflows/push.yaml

```diff
@@ -194,6 +194,71 @@ jobs:
           name: macos-sparkle-update
           path: ${{ github.workspace }}/output
 
+  create-windows-update:
+    name: Create Windows Update 🥩
+    if: github.repository_owner == 'obsproject' && github.ref_type == 'tag'
+    runs-on: windows-2022
+    needs: build-project
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+    defaults:
+      run:
+        shell: pwsh
+    environment:
+      name: bouf
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: "repo"
+          fetch-depth: 0
+          ref: ${{ github.ref }}
+
+      - name: Set Up Environment 🔧
+        id: setup
+        env:
+          BOUF_ACTION_HASH: '63933179a2cc636cb2bca2ae4ac2ae30af71bf29cc2a71f6ab01600a6f586bfe'
+        run: |
+          $shortHash = $env:GITHUB_SHA.Substring(0,9)
+          $channel = 'stable'
```

Comment:
Use PowerShell conditional expression:

```
$output = if ($env:GITHUB_REF_NAME -match "(beta|rc)") { "beta" } else { "stable" }
```

It's similar to pattern matching expressions.